### DISCUSSION TITLE
Bubble up errors on dependency graph failures

### DISCUSSION THREAD

**Discussion by OnCoreStudios** [supabase/supabase#29274] [Feature Requests]
Hi,

I recently upgraded and ran into `worker boot error: failed to create the graph` in a couple of my edge functions. I traced the issue down to my end where I needed to reinstall/update a package. Full context is here: https://discord.com/channels/839993398554656828/1283619547433402448

While this was a user error, I think we could possibly make the error messages more informative. I see the error message is coming from https://github.com/supabase/edge-runtime/blob/main/crates/sb_graph/graph_util.rs#L427 and there are more error messages like this one.

It looks like the functions return an `AnyError`. Does this error contain anything useful? If so, can we include it in the `.context`?

**Comment by linancn** [comment]
I encountered the same issue. It seems that some referenced TS files were not copied. The newer version of edge-function docker doesn’t specify which files are missing. You can use version 1.54.10, which will indicate exactly which reference files are missing.
